packages/cli-repl/src/format-output.ts

```diff
@@ -118,8 +118,12 @@ function formatSimpleType(output: any, options: FormatOptions): any {
   return inspect(output, options);
 }
 
-function formatCollections(output: string[], options: FormatOptions): string {
-  return clr(output.join('\n'), 'bold', options);
+function formatCollections(output: any[], options: FormatOptions): string {
```

Comment:
Ditto here, I think we want to avoid `any` wherever we can… maybe you can export the type from `@mongosh/shell-api`, the same way we do it for `HelpProperties`?